File: app/src/main/java/com/earth/OsToolkit/base/BaseJavaOperation.java
package com.earth.OsToolkit.base;
/*
 * OsToolkit - Kotlin
 *
 * Date : 11/1/2019
 *
 * By   : 1552980358
 *
 */

import android.app.ActivityManager;
import android.content.Context;

public class BaseJavaOperation {
    public static ActivityManager.MemoryInfo getMemory(Context context) {
        ActivityManager.MemoryInfo memoryInfo = new ActivityManager.MemoryInfo();
        ActivityManager activityManager = (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
        activityManager.getMemoryInfo(memoryInfo);
        return memoryInfo;
    }
}

File: app/src/main/java/com/earth/OsToolkit/Base/BaseJavaOperation.kt
package com.earth.OsToolkit.base

/*
 * OsToolkit - Kotlin
 *
 * Date : 11/1/2019
 *
 * By   : 1552980358
 *
 */

import android.app.ActivityManager
import android.content.Context

object BaseJavaOperation {
    fun getMemory(context: Context): ActivityManager.MemoryInfo {
        val memoryInfo = ActivityManager.MemoryInfo()
        val activityManager = context.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
        activityManager.getMemoryInfo(memoryInfo)
        return memoryInfo
    }
}
